package.json

```diff
@@ -124,6 +125,7 @@
     "jest": "^26.0.1",
     "jest-rdf": "^1.5.0",
     "manual-git-changelog": "^1.0.1",
+    "node-fetch": "^2.6.1",
```

Comment:
Would suggest using cross-fetch. It's also what we use in Comunica and can be useful should this ever need to run on a non-node platform for some reason.